src/components/Error404.tsx

```diff
@@ -0,0 +1,27 @@
+import Menu from './Menu';
```

Comment:
Make sure to import React in this file!

Reply:
It looks like the auto linting was removing the import. Should I still include it?

Reply:
That's weird.. How are you importing it? And make sure to specify that the `Error404` component is returning a `ReactElement` type.

Reply:
100% imported the wrong thing, thank you!